engine/src/flutter/.ci.yaml

```diff
@@ -472,6 +472,24 @@ targets:
       - os=Mac-14
       - cpu=x86
 
+  - name: Mac mac_ios_engine_ddm
+    recipe: engine_v2/engine_v2
+    timeout: 240
```

Comment:
when adding android, we were asked to also write `bringup: true` to ensure it was scheduled in the staging bots, but I'm not sure if that applies here.

@matanlurey would know :)

Reply:
If this is a new build, you will *have* to add `bringup: true`.

And yes, let's initially keep it as a bringup build.

Reply:
Done.